### PR TITLE
Update set-up-new-message-encryption-capabilities.md

### PR DESCRIPTION
**Pull request by msbemba**
added step to troubleshoot if step3 fails.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7886

File: microsoft-365/compliance/set-up-new-message-encryption-capabilities.md [hunk 1]
```diff
@@ -102,7 +102,15 @@ You can verify that your Microsoft 365 tenant is properly configured to use the
 
    - The default template names may be different from those displayed above. See [Configuring and managing templates for Azure Information Protection](/azure/information-protection/configure-policy-templates) for more.
 
-4. Run the Remove-PSSession cmdlet to disconnect from the Rights Management service.
+4.If the test fails with an error message **Failed to acquire RMS templates**, you need to execute the following commands and  Run the Test-IRMConfiguration cmdlet to verify it passes.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/set-up-new-message-encryption-capabilities.md]:
```suggestion
4. If the test fails with an error message **Failed to acquire RMS templates**, execute the following commands and  Run the Test-IRMConfiguration cmdlet to verify that it passes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

File: microsoft-365/compliance/set-up-new-message-encryption-capabilities.md [hunk 2]
```diff
@@ -102,7 +102,15 @@ You can verify that your Microsoft 365 tenant is properly configured to use the
 
    - The default template names may be different from those displayed above. See [Configuring and managing templates for Azure Information Protection](/azure/information-protection/configure-policy-templates) for more.
 
-4. Run the Remove-PSSession cmdlet to disconnect from the Rights Management service.
+4. If the test fails with an error message **Failed to acquire RMS templates**, execute the following commands and  Run the Test-IRMConfiguration cmdlet to verify that it passes.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/set-up-new-message-encryption-capabilities.md]:
```suggestion
4. If the test fails with an error message **Failed to acquire RMS templates**, execute the following commands and run the Test-IRMConfiguration cmdlet to verify that it passes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**